### PR TITLE
Setup the branch

### PR DESCRIPTION
**Pull request by jrraul4**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          </section>
+          <section class="main-section" id="id"><header>
+            Variable scope
+          </header>
+          <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Neque dolore reiciendis possimus quibusdam! Dolorem similique quibusdam tempora obcaecati quod, blanditiis vero debitis harum fugiat quaerat ipsum dolore, sed deleniti accusantium.</p>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis sint fuga cupiditate tenetur laboriosam magni alias facere totam optio earum nostrum, incidunt quas beatae. Nostrum quas quidem ipsa. Dolor, suscipit!</p>
+          <code>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas architecto voluptatum atque similique cupiditate, excepturi, debitis quibusdam deserunt expedita tempora aliquam temporibus in itaque magnam ut beatae molestiae quia eaque.</code>
+          <code>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Fuga quo quisquam ut alias veniam beatae suscipit nemo nisi quidem cupiditate omnis ad sapiente dicta ipsum iure vitae est, provident magnam.</code>
+          </section>
```
Comment (jrraul4) [inline index.html:74]: In this part, you could use only one section semantic tag to make your code cleaner 😉

File: style.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+*{
+    padding: 0;
+    margin: 0;
+
+}
+body{
+    display: flex;
```
Comment (jrraul4) [inline style.css:7]: Please use all possible Flexbox properties to give a better appearance to your web page 😎

File: style.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    padding: 10px;
+}
+main{
+    padding: 3rem;
+}
+main li{
+    list-style-position: inside;
+    text-indent: -1em;
+  margin-left: 3em;
+}
```
Comment (jrraul4) [inline style.css:42]: kindly add a media query to make your website responsive 👌

**Review by jrraul4 — COMMENTED** [review]
Hi @jrraul4 ,

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!



Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

Required Changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @jrraul4, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job making use of GitHub inline comments in your review :+1: 
- Nice work using a couple of emojis :1st_place_medal: 

## Required Changes ♻️
- [ ] Kindly rephrase your comments politely as suggestions, not an order. Using the word "Kindly", "Please", etc will help you sound more polite. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shahadat3669 — APPROVED** [review]
Hi @jrraul4 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Comments are polite, clear, and easy to understand 👍
✅ Add multiple comments 👍
✅ Used the GitHub inline comments feature 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**